crates/hyperswitch_interfaces/src/consts.rs

```diff
@@ -11,3 +11,6 @@ pub const ACCEPT_HEADER: &str = "text/html,application/json";
 
 /// User agent for request send from backend server
 pub const USER_AGENT: &str = "Hyperswitch-Backend-Server";
+
+/// Nomupay payout purpose
+pub const OTHER: &str = "OTHER";
```

Comment:
Keep this in `crates/hyperswitch_connectors/src/connectors/nomupay/transformers.rs`